### PR TITLE
#3994 [Manager] Topic element force delete

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3994)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/Pipeline.js [hunk 1]
```diff
@@ -61,7 +62,10 @@ const Pipeline = () => {
     data: currentStream,
     lastUpdated: streamLastUpdated,
   } = context.useStreamState();
-  const { lastUpdated: connectorLastUpdated } = context.useConnectorState();
+  const {
+    lastUpdated: connectorLastUpdated,
+    data: currentConnector,
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Pipeline.js]: Shouldn't this just be `connectors`?

File: ohara-manager/client/src/components/Pipeline/Pipeline.js [hunk 2]
```diff
@@ -537,6 +625,16 @@ const Pipeline = () => {
         data={PropertyDialogData}
         handleSubmit={handleSubmit}
       />
+      <Dialog
+        open={isOpen}
+        title={'Force Delete Topic?'}
+        children={
+          'You are sure to force the deletion of topic, This operation cannot be replied.'
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Pipeline.js]: We have a certain format for deletion message. And the object name (in this case: topicName) should be displayed as well

```js
Are you sure you want to delete the node: ${topicName} ? This action cannot be undone!
```

File: ohara-manager/client/src/components/Pipeline/Pipeline.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/Pipeline/Pipeline.js]: sure

File: ohara-manager/client/src/components/Pipeline/Pipeline.js (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/Pipeline/Pipeline.js]: Thanks

**Pull request merged**